packages/components/package.json

```diff
@@ -40,6 +40,7 @@
     "remark-gfm": "^4.0.0"
   },
   "peerDependencies": {
+    "@ttoss/react-hooks": "workspace:^",
```

Comment:
Change the docs to install `@ttoss/react-hooks` when installing `@ttoss/components`.